lbrynet/lbrynet_daemon/LBRYDaemonServer.py

```diff
@@ -179,12 +361,35 @@ def render_GET(self, request):
     #     call.addErrback(lambda err: log.info("Error: " + str(err)))
     #     call.cancel()
 
```

Comment:
being picky, but leaving two lines between classes and other top-level modules helps make the code more readable.

https://google.github.io/styleguide/pyguide.html#Blank_Lines